pyproject.toml

```diff
@@ -210,6 +212,7 @@ transformations = [
   "pykalman-bardo<0.10,>=0.9.7",
   "statsmodels<0.15,>=0.12.1",
   'stumpy<1.13,>=1.5.1; python_version < "3.12"',
+  'temporian',
```

Comment:
```suggestion
  'temporian<0.8,>=0.7.0; python_verion < "3.12"',
```

Reply:
I understood you proposed to remove the upper bound on the python version here? https://github.com/sktime/sktime/pull/5980#discussion_r1517981969 (couldn't find original comment)

Will add the bounds to the temporian version 👍🏼

Reply:
Here's the original comment: https://github.com/sktime/sktime/pull/5980#discussion_r1516312507. I suggested "only lower bound" for the `python_version` tag corresponding to the estimator itself. That suggestion still remains valid, and this current suggestion is for `pyproject.toml`.

Without these upper bounds on `temporian`, all tests in all of our PR installing these extras will fail on 3.12 as it'll fail to find `temporian` (as of now), and users directly using these extras will be affected as well. But with these, it'll simply ignore `temporian` for 3.12 and so rest of the estimators can be tested.

Reply:
Got it, my bad, wrongly remembered the original comment was on the pyproject :) Thanks!